File: Program17.c
#include<stdio.h>

void ReverseFactors(int iNo)
{
    int iCnt = 0 ;

    printf("Reverse Factors of Given Number are : \n") ;

    for(iCnt = iNo -1 ; iCnt >  1; iCnt-- )
    {
        if(iNo % iCnt == 0 )
        {
            printf("%d\t",iCnt) ;
        }
    }
}
int main()
{
    int iValue = 0 ;

    printf("Enter any Number\n") ;
    scanf("%d",&iValue) ;

    ReverseFactors(iValue) ;

    return 0 ;
}

File: Program17.py
# Accept a Number from user and display it's Even Factors 

def displayEvenFactors(iValue):

    for i in range(1,iValue):
        if( iValue % i == 0 and i % 2 == 0 ):
            print(i,end=" ")
            
def main():
    iNo = int(input("Enter a Number : ")) 

    displayEvenFactors(iNo)

if __name__=="__main__":
    main()